src/vs/workbench/contrib/chat/browser/chatContentParts/chatElicitationContentPart.ts

```diff
@@ -25,10 +25,10 @@ export class ChatElicitationContentPart extends Disposable implements IChatConte
 		@IInstantiationService private readonly instantiationService: IInstantiationService,
 	) {
 		super();
-
+		const { acceptButtonLabel, rejectButtonLabel } = elicitation;
 		const buttons = [
-			{ label: localize('accept', "Respond"), data: true },
-			{ label: localize('dismiss', "Cancel"), data: false, isSecondary: true },
+			{ label: acceptButtonLabel ?? localize('accept', "Respond"), data: true },
+			{ label: rejectButtonLabel ?? localize('dismiss', "Cancel"), data: false, isSecondary: true },
 		];
```

Comment:
FYI I think this will conflict with your change @meganrogge https://github.com/microsoft/vscode/pull/256613

@joshspicer is this just a future thinking change? Could bring it in via @meganrogge's if so and avoid the conflict.

Reply:
Ah nice! I'm using it in a branch but happy to adopt Megan's change